apps/backend/src/libs/modules/server-application/base-server-application.ts

```diff
@@ -28,9 +31,17 @@ type Constructor = {
 	config: Config;
 	database: Database;
 	logger: Logger;
+	options: Options;
```

Comment:
I meant that we don't need to pass them as separate options, let's pass them to the first level as other properties
```suggestion
	services: {
		userService: UserService;
	};
	token: Token;
```